README.md

```diff
@@ -22,8 +36,8 @@ Building The Provider
 ---------------------
 
 1. Clone the repository
-1. Enter the repository directory
-1. Build the provider using the Go `install` command:
+2. Enter the repository directory
+3. Build the provider using the Go `install` command:
```

Comment:
This isn't strictly necessary. Markdown automatically renders the correct count in numbered lists even if the entries in source use `1.`. 

The benefit of this for long lists is that if you remove one item, the remaining items do not need to be re-numbered in order for items to be displayed in sequential order.